tests/test_queries.py

```diff
@@ -196,3 +196,36 @@ def test_mapping_resolver_to_object_attribute():
     result = graphql(schema, "{ user { firstName } }")
     assert result.errors is None
     assert result.data == {"user": {"firstName": "Joe"}}
+
+
+def test_mapping_resolver_to_object_function():
```

Comment:
@salwator has converted me to story-telling test names, and we would like to move our tests suite to those. Perhaps `test_default_resolver`

I'm also thinking that test scenario could be more intuitive/real-worldly. Maybe we could have `logs: Logs` resolver accepting `env: String!`, asserting that it has valid value and returning Mock with `count` resolving to `get_items_count()`?